css-properties-values-api/Overview.bs

```diff
@@ -640,6 +640,9 @@ Parsing the syntax string {#parsing-syntax}
 		and set |component|’s |name| to the returned value.
 		Otherwise return failure.
 
+		If |component|’s |name| does not match the <<custom-ident>> production,
```

Comment:
```suggestion
		If |component|’s |name| does not [=CSS/parse=] as a <<custom-ident>>,
```

Reply:
Yeah, that makes sense. Thanks. 👍

Reply:
That means the following probably isn't great either:

> Any sequence which starts an identifier, can be consumed as a name, and matches the &lt;custom-ident> production

Reply:
Ah, indeed.